core/kazoo_csv/download_csvs.py

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env python2
-
-import requests, sys, json
-
-print("version: ", requests.__version__)
-csvs_url = 'https://api.github.com/repos/maxogden/csv-spectrum/contents/csvs'
```

Comment:
Attribution for 327 bytes of text?
1. Attribution is right there in our git history
2. These tests were never run by CI
3. These tests did not pass CI

I want them here so that they are actually used.